src/changelog.ts

```diff
@@ -51,22 +93,25 @@ function stringifyReleases(releases, changes) {
   return [stringifiedUnreleased, ...stringifiedReleases].join('\n\n');
 }
 
-function withTrailingSlash(url) {
+function withTrailingSlash(url: string) {
   return url.endsWith('/') ? url : `${url}/`;
 }
 
-function getCompareUrl(repoUrl, firstRef, secondRef) {
+function getCompareUrl(repoUrl: string, firstRef: string, secondRef: string) {
   return `${withTrailingSlash(repoUrl)}compare/${firstRef}...${secondRef}`;
 }
 
-function getTagUrl(repoUrl, tag) {
+function getTagUrl(repoUrl: string, tag: string) {
   return `${withTrailingSlash(repoUrl)}releases/tag/${tag}`;
 }
 
-function stringifyLinkReferenceDefinitions(repoUrl, releases) {
+function stringifyLinkReferenceDefinitions(
+  repoUrl: string,
+  releases: ReleaseMetadata[],
+) {
   const releasesOrderedByVersion = releases
     .map(({ version }) => version)
-    .sort((a, b) => {
+    .sort((a: string | semver.SemVer, b: string | semver.SemVer) => {
```

Comment:
The type for these values should be `Version` I think? I don't follow where `semver.SemVer` is coming from

```suggestion
    .sort((a: Version, b: Version) => {
```